### PR TITLE
WS: Marvel vs Capcom 2

### PR DESCRIPTION
**Pull request by RedDevilus**
image

Causes texture corruptions. (Europe ws patch)

### CODE REVIEW

**Comment by Mrlinkwii** [comment]
can you add why its  disabled in the patch  like //Causes texture corruptions

**Comment by refractionpcsx2** [comment]
Red can you action link's request please

File: cheats_ws/D14A6CD6.pnach [hunk 1]
```diff
@@ -1,12 +1,10 @@
-gametitle=Marvel vs Capcom 2 PAL (SLES_511.74)
-comment=Widescreen pnach by El_Patas
+//gametitle=Marvel vs Capcom 2 PAL (SLES_511.74)
+//comment=Widescreen pnach by El_Patas
```
Comment (Aced14) [inline cheats_ws/D14A6CD6.pnach:2]: These lines don't need to be commented-out. ``gametitle`` and ``comment`` are part of the pnach format's syntax.

```suggestion
gametitle=Marvel vs Capcom 2 PAL (SLES_511.74)
comment=Widescreen pnach by El_Patas
```

**Comment by RedDevilus** [comment]
It breaks main menu, not sure what's more to tell.

**Comment by refractionpcsx2** [comment]
Superseeded by #71

**Comment by RedDevilus** [comment]
Welp I just fixed it lol

**Pull request closed without merging**